training/cogvideox_image_to_video_lora.py

```diff
@@ -651,6 +651,8 @@ def load_model_hook(models, input_dir):
 
         for step, batch in enumerate(train_dataloader):
             models_to_accumulate = [transformer]
+            gradient_norm_before_clip = None
```

Comment:
Oh I thought we always calculate it?
https://github.com/a-r-r-o-w/cogvideox-factory/blob/6c00cf094b2f124a61775276d934e483b53b53fb/training/cogvideox_image_to_video_lora.py#L763